### PR TITLE
Ah/86bzf8u5x/load static data migrations

### PR DESCRIPTION
**Pull request by ahmedhamilastage**
-Removed LabeledIcon Model so changes have been made for following models : 
Need,PieceRenovate,ProjectExtension,ArchitectSpeciality,ArchitecturalStyle,ProjectCategory,PropertyType,WorkType,SupplierSpeciality 

-Added Fixtures yaml files , running the data loading example : 
image

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -17,9 +17,7 @@ psycopg2-binary = "2.9.9"
 whitenoise = "==6.6.0"
 twilio = "==9.2.0"
 djangorestframework-camel-case = "==1.4.2"
-pip = "*"
-install = "*"
-drf-standardized-errors = "*"
+pyyaml = "*"
```
Comment (MohamedMegdich1) [inline Pipfile:20]: why are you using pyyaml?

File: app/announcement/models/Need.py [hunk 2]
```diff
@@ -2,33 +2,45 @@
...
 """
 
 from django.db import models
 
-from app.core.models import LabeledIcon
 from app.core.models.ArchitectSpeciality import ArchitectSpeciality
 
 
-class Need(LabeledIcon):
+class Need(models.Model):
```
Comment (MohamedMegdich1) [inline app/announcement/models/Need.py:13]: why are you removing the LabeledIcon class?

File: app/announcement/models/Need.py [hunk 3]
```diff
@@ -2,33 +2,45 @@
...
-        LabeledIcon: Base class providing fields for label and icon.
 
     Attributes:
         architect_speciality (ForeignKey): Specialization of the architect related to this need.
+        label
+        icon
     """
 
+    label = models.CharField(max_length=255, default="")
+    icon = models.ImageField(upload_to="NeedIcons/")
```
Comment (MohamedMegdich1) [inline app/announcement/models/Need.py:26]: You should specify a folder for this type of icon and then split it depending on the usage

**Review by MohamedMegdich1 — CHANGES_REQUESTED** [review]

File: Pipfile (hunk 1 above)
Comment (ahmedhamilastage) [reply Pipfile:20]: Django uses it as dependency to load data in yaml format

File: app/announcement/models/Need.py (hunk 2 above)
Comment (ahmedhamilastage) [reply app/announcement/models/Need.py:13]: In inserting data for models inheriting from LabeledIcon it becomes unnecessarily complicated, mist add instance for LabeledIcon and then create child instance and point to it , and when all fixtures depend from LabeledIcon there can be conflicts if ptr values overlapse

File: app/announcement/models/Need.py (hunk 3 above)
Comment (ahmedhamilastage) [reply app/announcement/models/Need.py:26]: Can you explain more please, the icon images are uploaded to media/NeedIcons , where should they be uploaded?

**Pull request closed without merging**